main.tf

```diff
@@ -1,13 +1,112 @@
-#
-# Developer tips:
-#   - Below code should be replaced with the code for the root level module
-#
-
-resource "ibm_resource_instance" "cos_instance" {
-  name              = var.name
-  resource_group_id = var.resource_group_id
-  service           = "cloud-object-storage"
-  plan              = var.plan
-  location          = "global"
-  tags              = var.resource_tags
+##############################################################################
+# RedHat OpenShift Service Mesh 3
+# Deploy the Service Mesh operator on an OCP cluster and sets up
+# one or several service mesh control plane(s)
+##############################################################################
+
+##############################################################################
+# Locals
+##############################################################################
+
+locals {
+  operators_namespace      = "openshift-operators"
+  sm_operator_release_name = "helm-release-smv3-subscription"
+  sm_operator_chart_path   = "servicemeshoperator"
+  sm_operator_version      = "v3.0.3"
```

Comment:
We need a way to manage this version - suggest to create a tracking issue and we can see if we can use renovate